google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/readrows/ReadRowsRetryTest.java

```diff
@@ -322,6 +433,8 @@ public void readRows(
       }
       if (expectedRpc.statusCode.toStatus().isOk()) {
         responseObserver.onCompleted();
+      } else if (expectedRpc.statusException != null) {
```

Comment:
I don't think you need to create a different list. Just add them to expectedRpc.exception. Because your largeRowException is returning ApiExceptions